### DISCUSSION TITLE
ETA for socket.io-related refactoring

### DISCUSSION THREAD

**Discussion by mrora** [eclipse-theia/theia#10660] [General] [answered]
Hello, team!

There is a PR (https://github.com/eclipse-theia/theia/pull/10514) for refactoring the websocket-related communication logic to use socket.io to address #10403. I am curious if there is an upcoming release this change is expected to land in.

Thanks!

**Comment by msujew** [comment, marked as answer]
Hi @mrora,

I'm glad you're interested in that feature/improvement :)

I haven't found the right abstraction for that yet and the team is mainly trying to land #9936, so I am optimistically looking for the February Release (1.23).
